doc/whats_new/v0.23.rst

```diff
@@ -316,6 +316,10 @@ Changelog
   ``max_value`` and ``min_value``. Array-like inputs allow a different max and min to be specified
   for each feature. :pr:`16403` by :user:`Narendra Mukherjee <narendramukherjee>`.
 
+- |Enhancement| :class:`impute.SimpleImputer`, :class:`impute.KNNImputer`, and
+  :class:`impute.SimpleImputer` accepts pandas IntegerArray with nan values.
```

Comment:
```suggestion
  :class:`impute.SimpleImputer` accepts pandas' nullable integer dtype with missing values.
```

Just a suggestion, but in general we try to speak about the dtype, as "IntegerArray" is only used under the hood and not directly visible to most users (unless you access the underlying values of a single column)

Reply:
(but it makes it more verbose, though, ..)

Reply:
@jorisvandenbossche that's a good point. There are other occurrence of the "IntegerArray" in this PR. I think we should covert them all to "pandas' nullable integer dtype/values/column/dataframe" depending on the context.

Reply:
@thomasjpfan if you agree with this proposal, I can give it a shot.

Reply:
I agree.

Reply:
I updated the PR with removing the "IntegerArray" wording in place of "pandas' nullable ..."